app/views/govuk_publishing_components/components/docs/chart.yml

```diff
@@ -243,9 +243,10 @@ examples:
             - 80
   minimal_version:
     description: |
-      The minimal version presents a simplified version of the chart and should only be used where the user is then directed to a more detailed view of the data. Specifically, minimal mode:
+      The minimal version presents a simplified version of the chart. This should only be used where there is not enough space to display a full chart, and the user is then given an option to see more information about the chart, for example by including a link next to the chart component. This must be part of the page, as this is not provided by the component itself.
+
+      Specifically, minimal mode:
 
-      - turns the chart into a link that should point to a page with a full version of the chart with all data
       - hides the chart heading, but uses the text as the text for the link
```

Comment:
Does the reference to a link need to be removed on this line as well?

Reply:
Good spot, have removed, and added a changelog entry. Thanks! 👍

Reply:
@AshGDS are you happy to approve the visual diff?

Reply:
Yep, done 👍